Source/AwsCommonRuntimeKit/http/HTTPClientConnectionManager.swift

```diff
@@ -22,7 +22,7 @@ public class HTTPClientConnectionManager {
     }
 
     /// Acquires an `HTTPClientConnection` asynchronously.
-    public func acquireConnection() async throws -> HTTPClientConnection {
+    public func acquireConnection() async throws -> HTTPExchange {
```

Comment:
acquireConnection() returns an HTTPExchange but releaseConnection() takes an HTTPClientConnection

stop using exchange for now, or make them both use exchange (my preference is stop using "exchange")

Reply:
Thanks, fixed.